### PR TITLE
add shadcn

### PR DESCRIPTION
**Pull request by Lev1ty**

### CODE REVIEW

File: web/app/globals.css
Comment (Zack-Ren) [inline web/app/globals.css:1]: Does this override nextui styling?

Comment (Lev1ty) [reply web/app/globals.css:1]: check in tailwind config

Comment (Lev1ty) [reply web/app/globals.css:1]: i think the lines removed were from the initial create next app, do not think nextui adds anything to globals css

File: web/lib/utils.ts
Comment (Zack-Ren) [inline web/lib/utils.ts:1]: is there something like `bazel lint` so we can avoid this issue

Comment (Lev1ty) [reply web/lib/utils.ts:1]: found https://github.com/aspect-build/rules_lint, but does not seem to support biome

Comment (Lev1ty) [reply web/lib/utils.ts:1]: my current workflow has biome in editor fix on save and then biome ci to verify
this does miss generated files though on local

Comment (Zack-Ren) [reply web/lib/utils.ts:1]: would a formatter within the IDE work?

Comment (Lev1ty) [reply web/lib/utils.ts:1]: should work https://biomejs.dev/reference/vscode/, i think for vscode if you set it as formatter it formats on save

Comment (Zack-Ren) [reply web/lib/utils.ts:1]: yeah that looks good

**Review by Zack-Ren — APPROVED** [review]

**Review by sunb26 — APPROVED** [review]

**Pull request merged**